notesserver/settings/common.py

```diff
@@ -131,14 +129,10 @@
 JWT_AUTH = {
     'JWT_AUTH_HEADER_PREFIX': 'JWT',
     'JWT_ISSUER': [
-        {
-            'AUDIENCE': 'SET-ME-PLEASE',
-            'ISSUER': 'http://127.0.0.1:8000/oauth2',
-            'SECRET_KEY': 'SET-ME-PLEASE'
-        }
+        {'AUDIENCE': 'SET-ME-PLEASE', 'ISSUER': 'http://127.0.0.1:8000/oauth2', 'SECRET_KEY': 'SET-ME-PLEASE'}
```

Comment:
ditto

Reply:
I like trailing commas it a list too. Safety is our priority 🙂